desktop_server/src/com/mygdx/game/server/network/Server.java

```diff
@@ -78,13 +85,18 @@ private void handle(
           var buffer = Buffer.buffer(json.toJson(msg, (Class<?>) null));
           ws.getSocket().write(buffer);
         });
+        endTurnService.giveTurnToFirstPlayer();
```

Comment:
może da się połączyć GameStartedMessage i to?

Reply:
no bo tutaj widać dwa te nasze dwa sposoby komunikacji - bezpośredniej przez deserializacje obiektów Message i tej wyspecjalizowanej przez te ComponentMessage

GameStartedMessage jest obecnie nadmiarowe, albo to giveTurnToFirstPlayer jest nadmiarowe